etc/openqa/openqa.ini

```diff
@@ -115,6 +115,8 @@
 #do_push = no
 ## whether to clone CASEDIR or NEEDLES_DIR on the web UI if that var points to a Git repo
 #git_auto_clone = yes
+## Experimental - Ensure a git update of all test code and needles regardless of test execution
```

Comment:
Ensure a *periodic* git update?

Reply:
I changed it now and made it shorter.
Can we talk about doc improvements later please?
We should really get the feature done and deployed in production and see how it performs.